src/redux/message-service/saga.ts

```diff
@@ -116,6 +116,9 @@ function* receiveFromFirebaseSaga(address) {
     const message = yield take(channel);
     // tslint:disable-next-line:no-console
     console.log('[MESSAGE SAGA] message received');
+
+    yield delay(500);
```

Comment:
It'd be nice to be able to easily flag this on and off based on an environment variable for debugging purposes.